.github/workflows/matlab.yml

```diff
@@ -95,6 +95,7 @@ jobs:
           MATLABPATH: matlab/install/arrow_matlab
         uses: matlab-actions/run-tests@v2
         with:
+          strict: true
           select-by-folder: matlab/test
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Will do!